FinniversKit/Sources/Components/SideScrollableView/SideScrollableView.swift

```diff
@@ -0,0 +1,188 @@
+                     labelHeight +
+                     SideScrollableView.contentInset.bottom
+        return CGSize(width: UIView.noIntrinsicMetric,
+                      height: height)
+    }
+
+    public weak var delegate: SideScrollableViewDelegate?
+    
+    // MARK: - Init
+    
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+    
+    // MARK: - Setup
+    
+    public func configure(with viewModel: SideScrollableItemModel) {
+        cleanup()
+        createButtons(withTitles: viewModel.items)
+        subscribeToButtonTaps()
+    }
+
+    private func setup() {
+        addSubview(scrollView)
+        scrollView.addSubview(indicatorView)
+
+        // Bind scrollView to the view boundaries
+        NSLayoutConstraint.activate([
+            scrollView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            scrollView.topAnchor.constraint(equalTo: topAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ])
+        
+        // Bind the stackView to the scroll view's content layout guide to
+        // set the content size.
+        NSLayoutConstraint.activate([
+            contentView.leadingAnchor.constraint(equalTo: scrollView.contentLayoutGuide.leadingAnchor),
+            contentView.topAnchor.constraint(equalTo: scrollView.contentLayoutGuide.topAnchor),
+            contentView.trailingAnchor.constraint(equalTo: scrollView.contentLayoutGuide.trailingAnchor),
+            contentView.bottomAnchor.constraint(equalTo: scrollView.contentLayoutGuide.bottomAnchor)
+        ])
+        
+        indicatorViewLeadingConstraint = indicatorView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor)
+        indicatorViewWidthConstraint = indicatorView.widthAnchor.constraint(equalToConstant: 0)
+        
+        NSLayoutConstraint.activate([
+            indicatorViewLeadingConstraint,
+            indicatorView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            indicatorViewWidthConstraint
+        ])
+    }
+
+    private func cleanup() {
+        cancellables.removeAll()
+        buttons.removeAll()
+        contentView.removeArrangedSubviews()
+    }
+    
+    private func createButtons(withTitles itemTitles: [String]) {
+        // The current implementation always makes the first item as selected
+        for (index, itemName) in itemTitles.enumerated() {
+            let button = Button.makeSideScrollableButton(withTitle: itemName)
+            contentView.addArrangedSubview(button)
+            buttons.append(ButtonItem(title: itemName,
+                                      button: button))
+
+            button
+                .publisher(for: .touchUpInside)
+                .sink { [weak self, index] _ in
```

Comment:
`index` is an Int so it should already be captured by this closure without the placing it in this list, right?